presto-verifier/src/test/java/com/facebook/presto/verifier/VerifierTestUtil.java

```diff
@@ -55,31 +57,12 @@ public static StandaloneQueryRunner setupPresto()
         return queryRunner;
     }
 
-    public static String getJdbcUrl(StandaloneQueryRunner queryRunner)
-    {
-        return queryRunner.getServer().getBaseUrl().toString().replace("http", "jdbc:presto");
-    }
-
     public static TestingMySqlServer setupMySql()
             throws Exception
     {
         TestingMySqlServer mySqlServer = new TestingMySqlServer("testuser", "testpass", ImmutableList.of(XDB), MY_SQL_OPTIONS);
         try (Handle handle = getHandle(mySqlServer)) {
-            handle.execute("CREATE TABLE verifier_queries (\n" +
-                    "  id int(11) unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,\n" +
-                    "  suite varchar(256) NOT NULL,\n" +
-                    "  name varchar(256) DEFAULT NULL,\n" +
-                    "  control_catalog varchar(256) NOT NULL,\n" +
-                    "  control_schema varchar(256) NOT NULL,\n" +
-                    "  control_query text NOT NULL,\n" +
-                    "  test_catalog varchar(256) NOT NULL,\n" +
-                    "  test_schema varchar(256) NOT NULL,\n" +
-                    "  test_query text NOT NULL,\n" +
-                    "  control_username varchar(256) NOT NULL DEFAULT 'verifier-test',\n" +
-                    "  control_password varchar(256) DEFAULT NULL,\n" +
-                    "  test_username varchar(256) NOT NULL DEFAULT 'verifier-test',\n" +
-                    "  test_password varchar(256) DEFAULT NULL,\n" +
-                    "  session_properties_json varchar(2048) DEFAULT NULL)");
+            handle.attach(VerifierDao.class).createVerifierQueriesTable(new MySqlSourceQueryConfig().getTableName());
```

Comment:
@caithagoras How are you deciding which queries belong to DAO and which do not?

Reply:
- Queries that are used in non-test code goes into DAO.
- CREATE TABLE queries that creates the table used by non-test code should also goes into DAO, so that they're closer to the usages and are easier for reference.